docs/docsite/rst/network/user_guide/network_resource_modules.rst

```diff
@@ -0,0 +1,158 @@
+.. _resource_modules:
+
+************************
+Network resource modules
+************************
+
+Ansible 2.9 introduced network resource modules to simplify and standardize how you manage different network devices.
+
+
+.. contents::
+   :local:
+
+Understanding network resource modules
+=======================================
+
+Network devices separate configuration into sections (such as interfaces, VLANS, etc) that apply to a network service. Ansible network resource modules take advantage of this to allow you to configure subsections or *resources* within the network device configuration. Network resource modules provide a consistent experience across different network devices.
+
+
+Network resource module states
+===============================
+
+You use the network resource modules by assigning a state to what you want the module to do. The resource modules support the following states:
+
+merged
+  Ansible merges the on-device configuration with the provided configuration in the task.
+
+replaced
+  Ansible replaces the on-device configuration subsection with the provided configuration subsection in the task.
+
+overridden
+  Ansible overrides the entire on-device configuration with the provided configuration in the task. Use caution with this state as you could remove your access to the device (for example, by overriding the management interface configuration).
```

Comment:
This overrides the configuration for that is managed by particular resource module and not the entire on-device running configuration